supplier_app/tests/test_view.py

```diff
@@ -124,9 +124,20 @@ def tearDown(self):
         ):
             rmtree('file')
 
-    def _get_example_forms(self):
+    def _get_example_forms(self, **kwargs):
+        eb_entities = []
+        if kwargs:
```

Comment:
Should we check that kwargs contains `eb_entities`?

Reply:
done